website/cue/reference/releases/0.43.1.cue

```diff
@@ -0,0 +1,41 @@
+package metadata
+
+releases: "0.43.1": {
+	date:     "2024-12-10"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		This patch release contains fixes for regressions in 0.43.0.
+		"""
+
+	changelog: [
+		{
+			type: "fix"
+			description: """
+				Update to VRL v0.20.1 which a reverts to previous `to_float` behavior for non-normal floats.
```

Comment:
I could also just copy paste the v0.20.1 changelog here

Reply:
True, that would be reasonable.